mypy/stubgen.py

```diff
@@ -222,25 +226,141 @@ def generate_stub(path: str, output_dir: str, _all_: Optional[List[str]] = None,
 NOT_IN_ALL = 'NOT_IN_ALL'
 
 
+class AnnotationPrinter(TypeStrVisitor):
+
+    def __init__(self, stubgen: 'StubGenerator') -> None:
+        super().__init__()
+        self.stubgen = stubgen
+        self.requires_quotes = False
+
+    def visit_unbound_type(self, t: UnboundType)-> str:
+        s = t.name
+        base = s.split('.')[0]
+        self.stubgen.import_tracker.require_name(base)
```

Comment:
when you have an `X.Y.Z` annotatation, what other things could `X` be? I'm assuming module here because if it's not the only choice, it's probably the best one.

Reply:
I am still not sure it is a good idea to guess something for an unbound type. I still think it would be better to use ``Any``. Why do you think we should do something at all with unbound types?

Reply:
Stubgen runs mostly at a syntactical level. Semantic analysis phase is not run within it. So essentially 99% of the type annotations it will find will still be unbound

Reply:
> Stubgen runs mostly at a syntactical level. Semantic analysis phase is not run within it.

OK, is it easy to fix this? If not, then maybe add a `` # TODO:`` item here and/or open an issue?

Reply:
I would also have an issue for this. Maybe some day we will have time to improve this.